end-to-end-tests/docker-compose.yaml

```diff
@@ -0,0 +1,85 @@
+version: '2.4'
+services:
+  anchor-platform-server:
+    image: anchor-platform
+    build:
+      context: ../../
+      dockerfile: integration-tests/Dockerfile
+    command: "--sep-server"
+    environment:
+      - STELLAR_ANCHOR_CONFIG=file:/config/anchor-platform-config.yaml
+      - SECRET_SEP10_SIGNING_SEED=${SECRET_SEP10_SIGNING_SEED?err}
+      - SECRET_SEP10_JWT_SECRET=secret
+      - SECRET_PLATFORM_API_AUTH_SECRET=myAnchorToPlatformSecret
+      - SECRET_CALLBACK_API_AUTH_SECRET=myPlatformToAnchorSecret
+      - SECRET_DATA_USERNAME=postgres
+      - SECRET_DATA_PASSWORD=password
+      - LOG_APPENDER=console_appender
+    volumes:
+      # add mounts for the new config directory
+      - ./anchor-platform-default-configs:/config_defaults
+    tmpfs:
+      - /config
+    depends_on:
+      - db
+      - kafka
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+
+  stellar-observer:
```

Comment:
The `stellar-observer` is also needed in [sep24 docker-compose file](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/docker-compose/anchor-get-started/sep24/docker-compose.yaml). 

Is there any way to share this configuration so that we don't have to duplicate the same configuration.

Reply:
Updated the common-services file to include the stellar-observer.